app/views/curation_concerns/base/_relationships_collection_row.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
+<th><%= t(".#{model_name}", type: model_name) %></th>
+<td>
+  <% if collection.blank? %>
+    <p><%= t('.empty', type: model_name) %></p>
+  <% else %>
+    <ul class="tabular">
+      <% collection.each do |col| %>
```

Comment:
collection in this sense isn't necessarily a `Collection`, so the variable naming here probably needs to be changed to maybe `list` and `item` when iterating. The type for a collection, is a grouping of `Collection`, child works, or parent works. The result is in the show page on the relationships table you would see a row with a list of all of the `Collection`, Children, and Parents.